rtb-house/DOCUMENTATION.md

```diff
@@ -2,56 +2,106 @@
 ## General Compatibility
 |Feature|  |
 |---|---|
-| Consent |  |
-| Native Ad Support |  |
-| SafeFrame Support |  |
-| PMP Support | |
+| Consent | Yes |
+| Native Ad Support | No |
```

Comment:
Index tego nie wspiera, nie?

Reply:
Chyba wspierają, niektórzy partnerzy zaznaczyli 'Yes'.
Chociaż nie widzę w kodzie adaptera tych partnerów, żeby występowało słowo 'native' później (spodziewam się, że do tej formy jest coś potrzebne inaczej - czy to tylko w endpoint'cie [server-side] się przestawia?)